api/stats.js

```diff
@@ -122,6 +129,8 @@ export class Stats {
         and post the weekly stats
     */
     postWeeklyStats = async () => {
+        // TODO: test this works or weekly
+        await this._iniClient();
```

Comment:
we tested daily stats, but this one was failing for some reason, we need to test weekly updates